File: 코딩 기초 트레이닝/I로 만들기-p181834/I로 만들기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181834

def solution(myString):
    ASCII_I = 108
    answer = ''

    myString = list(myString)

    for m in myString:
        if ord(m) < ASCII_I:
            answer += 'l'
        else:
            answer += m

    return answer

File: 코딩 기초 트레이닝/I로 만들기-p181834/I로 만들기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181834

function solution(myString) {
    const ASCII_I = 108
    myString = myString.split("").map((m) => {
        if(m.charCodeAt(0) < ASCII_I) {
            return 'l';
        } else {
            return m;
        }
    });

    return myString.join('');
}
